activerecord/lib/active_record/locale/en.yml

```diff
@@ -9,6 +9,7 @@ en:
     messages:
       required: "must exist"
       taken: "has already been taken"
+      record_not_destroyed: "Failed to destroy %{model} with %{key}"
```

Comment:
This should be inside the active_record configurations. It doesn't happen on Active Model